aemedge/blocks/columns/columns.css

```diff
@@ -395,6 +391,10 @@
       background-color: transparent;
     }
 
+    .columns > div > div:not(.columns-img-col){
+      place-self: center;
```

Comment:
You can remove all this, they are not aligning anything to the middle on the live site.